addons/kudo-kubeflow/0.x/kubeflow

```diff
@@ -0,0 +1,40 @@
+apiVersion: kubeaddons.mesosphere.io/v1beta1
+kind: ClusterAddon
+metadata:
+  name: kubeflow
+  labels:
+    kubeaddons.mesosphere.io/name: kubeflow
+    kubeaddons.mesosphere.io/hack-requires-defaultstorageclass: "true"
+  annotations:
+    catalog.kubeaddons.mesosphere.io/skip-namespace-creation: "true"
+    kudo.kubeaddons.mesosphere.io/create-namespace: "true"
+    catalog.kubeaddons.mesosphere.io/addon-revision: "1.0.1-1"
+    appversion.kubeaddons.mesosphere.io/kubeflow: "1.0.1"
+spec:
+  namespace: kubeflow
+  kubernetes:
+    minSupportedVersion: v1.16.0
+  requires:
+    - matchLabels:
+        kubeaddons.mesosphere.io/name: istio
+    - matchLabels:
+        kubeaddons.mesosphere.io/name: dex
+    - matchLabels:
+        kubeaddons.mesosphere.io/name: cert-manager
+    - matchLabels:
+        kubeaddons.mesosphere.io/name: knative
+  cloudProvider:
```

Comment:
Out of curiosity, why do we list AWS and Azure but not GCP?

Reply:
Ah, because the Konvoy on GCP support is still WIP